sdk/lib/drivers/sound/mmixer/wave.c

```diff
@@ -614,6 +614,41 @@ MMixerGetWaveOutCount(
     return MixerList->WaveOutListCount;
 }
 
+MIXER_STATUS
+MMixerGetWavePosition(
+    _In_ PMIXER_CONTEXT MixerContext,
+    _In_ HANDLE PinHandle,
+    _Out_LPDWORD Position)
+{
+    KSAUDIO_POSITION AudioPosition;
+    KSPROPERTY Property;
+    MIXER_STATUS Status;
+    ULONG Length;
+
+    /* Validate mixer context */
+    Status = MMixerVerifyContext(MixerContext);
+
+    if (Status != MM_STATUS_SUCCESS)
+        return Status;
+
+    Property.Id = KSPROPERTY_AUDIO_POSITION;
+    Property.Set = KSPROPSETID_Audio;
+    Property.Flags = KSPROPERTY_TYPE_GET;
+
+    Status = MixerContext->Control(PinHandle, IOCTL_KS_PROPERTY,
+                                   &Property, sizeof(KSPROPERTY),
+                                   &AudioPosition, sizeof(KSAUDIO_POSITION),
```

Comment:
```suggestion
                                   &Property, sizeof(Property),
                                   &AudioPosition, sizeof(AudioPosition),
```
Not hardcoding the types of the variables on which you do the sizeof

Reply:
Committed.